### PR TITLE
Adds query depth limit env var.

### PR DESCRIPTION
**Pull request by ashfurrow**
Env var name is `QUERY_DEPTH_LIMIT`.

### CODE REVIEW

**Comment by orta** [comment]
Yep 👍

**Comment by joeyAghion** [comment]
Thanks. Great addition!

File: index.js [hunk 1]
```diff
@@ -20,10 +20,11 @@ import moment from "moment"
 import * as tz from "moment-timezone" // eslint-disable-line no-unused-vars
 global.Promise = Bluebird
 
-const { PORT, NODE_ENV, GRAVITY_API_URL, GRAVITY_ID, GRAVITY_SECRET } = process.env
+const { PORT, NODE_ENV, GRAVITY_API_URL, GRAVITY_ID, GRAVITY_SECRET, QUERY_DEPTH_LIMIT } = process.env
 
 const app = express()
 const port = PORT || 3000
+const queryLimit = parseInt(QUERY_DEPTH_LIMIT, 10) || 10 // Default to ten.
```
[the remark this answers is not in the document]
Comment (orta) [reply index.js]: 10 is pretty far over the limit that we've seen in our usage ( [papertrail](https://papertrailapp.com/systems/artsy-metaphysics-staging/events?q=Depths%3A&focus=854492230537412638) ) - I think it's a safe enough default

[the remark this answers is not in the document]
Comment (alloy) [reply index.js]: I don’t think 10 is a bad default, but I would prefer that this only gets enabled when somebody is baby sitting this specific feature. Right now somebody would have to think of tailing the logs when they deploy and look out for this log.

**Pull request merged**